index.js

```diff
@@ -5,24 +5,24 @@ const links = document.querySelector("#links");
 const new_bg = document.querySelector("#new_bg");
 
 ham_menu.addEventListener("click", () => {
-    ham_menu.style.display = "none";
+  ham_menu.style.display = "none";
   ham_menu.classList.toggle("active");
   links.classList.toggle("active");
-    mobile_container.style.display = 'block';
+  mobile_container.style.display = "block";
 });
 
 document.querySelectorAll(".nv").forEach((n) =>
   n.addEventListener("click", () => {
     ham_menu.classList.remove("active");
-      ham_menu.style.display = "block";
+    ham_menu.style.display = "block";
     links.classList.toggle("active");
     new_bg.classList.toggle("active");
   })
 );
```

Comment:
Kindly ensure that the pop-up menu design matches that of the figma at all screen sizes of the mobile version since there are some slight differences between them currently at certain screen widths as shown below :point_down:  . 

- [ ] The dotted image below the close button on the top right side shifts to the middle at larger mobile widths around 600px


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-21 09-25-53](https://user-images.githubusercontent.com/65233860/203016783-c184c46c-69fe-4c86-b84c-e8fa4398ad39.png)  |  ![Screenshot from 2022-11-21 09-25-16](https://user-images.githubusercontent.com/65233860/203016861-11dbe1d4-0c38-46d4-bbd9-2b83485e55b4.png)





- [ ] There is a blank white space on the bottom and right side of the bottom image which increases with increasing mobile screen widths 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-21 09-25-26](https://user-images.githubusercontent.com/65233860/203017486-e16cf671-be38-47d4-9e83-bfd040c8748c.png)  |  ![Screenshot from 2022-11-21 09-25-07](https://user-images.githubusercontent.com/65233860/203017523-9554255e-3125-4b01-ab94-e4767c5c585d.png)



 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-21 09-26-08](https://user-images.githubusercontent.com/65233860/203017176-5d4659e8-0563-427b-b1fe-fb9789beb3ba.png)  |  ![Screenshot from 2022-11-21 09-26-15](https://user-images.githubusercontent.com/65233860/203017209-c1470e30-1e03-403d-b0a3-9fe20c31df0e.png)